scripts/generate-api.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+
+	"github.com/adeptmind/adept-go-postgres-api-boilerplate/internal/config"
```

Comment:
I'm not too happy about having the folder name here

Reply:
you could use the raw dot env operations to remove this issue